src/index.css

```diff
@@ -24,3 +24,16 @@ body {
 .search-row input {
   margin-right: 10px;
 }
+.wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+img {
+  width: 200px;
+  height: 250px;
+}
+th td {
+  border: 1px solid gray;
+}
```

Comment:
This works for a small app, but usually you would want a separate css file for each component.